.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
+- [ ] Bump Spec version.
+- [ ] Update last modified date.
+- [ ] Update changelog.
+- [ ] Make sure there is generated JSON files from the YAML test files.
+- [ ] Test changes in your language driver.
+
+Please review the [spec process wiki page](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=80806719) for additional information.
```

Comment:
In contrast to the "thanks again" comments, _this_ would be more suitable to include as a comment:

```suggestion
<!-- See also: https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=80806719 -->
```

I removed the title to be concise, but an alternative would be to have @agolin95 give that page a name so the URL would be self-describing.